app/controllers/spree/checkout_controller_decorator.rb

```diff
@@ -30,8 +30,4 @@ def before_address
     @order.bill_address ||= preferred_bill_address || last_used_bill_address || Spree::Address.default
     @order.ship_address ||= preferred_ship_address || last_used_ship_address || nil
   end
-
-  def after_complete
-    reset_order
-  end
```

Comment:
As discussed in Slack let's merge this since it will pass through the build pipeline as a whole with step 6.